guide/savings-wallet/miniscript-wallet-simple.md

```diff
@@ -0,0 +1,385 @@
+      alt:
+    - file: device-replacement/replacement-wallet-settings
+      alt:
+    - file: device-replacement/replacement-wallet-keys
+      alt:
+    - file: device-replacement/replacement-key-details
+      alt:
+    - file: device-replacement/replacement-use-case
+      alt:
+    - file: device-replacement/replacement-pairing-flow
+      alt:
+    - file: device-replacement/replacement-confirmation
+      alt:
+    - file: device-replacement/replacement-keys-updated
+      alt:
+---
+
+<!--
+
+Editor's notes
+
+This page covers a multi-key wallet that uses timelocks to provide additional recovery options.  
+
+Illustration sources
+
+https://www.figma.com/file/uxZMdWR2HfnKSMC9YWyr2h/Time-based-recovery?type=design&node-id=5046%3A3147&mode=design&t=5OZsga2vPqttH4Pl-1
+
+-->
+
+{% include picture.html
+   image = "/assets/images/guide/miniscript-wallet-simple/cover.png"
+   retina = "/assets/images/guide/miniscript-wallet-simple/cover@2x.png"
+   mobile = "/assets/images/guide/miniscript-wallet-simple/cover-mobile.png"
+   mobileRetina = "/assets/images/guide/miniscript-wallet-simple/cover-mobile@2x.png"
+   alt-text = "Image showing a wallet with recovery key sets"
+   caption = ""
+   width = 1600
+   height = 600
+   layout = "full-width"
+%}
+
+# Time-based recovery 
+{: .no_toc }
+
+---
+
+<div class="glossary-toc" markdown="1">
+ * Table of contents
+{:toc}
+</div>
+
+---
+
+In this reference design, we look at a variation of the [savings wallet reference design](https://bitcoin.design/guide/savings-wallet/). Specifically, we design a wallet application that allows users to enable a simple recovery path in order to provide even more fault tolerance than a regular multi-key wallet. 
+
+### Prior reading
+
+The use case for this design is relatively simple. However, we will be referencing some advanced bitcoin concepts and capabilities that are provided by [Miniscript](https://bitcoin.design/guide/glossary/#miniscript). If you are unfamiliar with the topic, please read the [custom spending conditions page](https://bitcoin.design/guide/how-it-works/custom-spending-conditions/) before diving deeper into this reference design.
+
+### Use case
+
+A multi-key wallet offers a number of advantages, especially for storing large amounts of bitcoin. Therefore, Bob and Alice, want to create a 2-of-3 multi-key wallet for their savings. Bob will hold the first key while his wife, Alice, will hold a second key. In addition, they have the third key locked away in a safety deposit box. 
+
+However, the couple worries about a situation where they lose access to two of the keys. In that case, they would lose all of their savings. To protect themselves against this scenario, our application offers them the possibility to enable a recovery path, which automatically lowers the amount of signatures needed to 1-of-3 after 6 months. This will allow them to lose two keys and still be able to recover their savings. 
```

Comment:
Good point. Losing a key completely implies that you lose the device plus the seed phrase backup. Yeah, I'd formulate it like this, probably:

"This will allow them to lose two keys as well as their respective seed phrase backups and still be able to recover their savings."

What do you think?

Reply:
That reads well. Go for it.